opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -680,7 +705,10 @@ relaxationFactorFractionsProducer(const BVectorWell& dwells) const
                 relaxation_factor *= further_relaxation_factor;
             }
         }
-        assert(relaxation_factor >= 0.0 && relaxation_factor <= 1.0);
+        if (relaxation_factor < 0.0 || relaxation_factor > 1.0) {
+            const std::string msg = fmt::format(" illegal relaxation_fractor {} is obtained for well {}", relaxation_factor, this->well_.name());
```

Comment:
Typo: fractor -> factor. Also, avoid variable names in messages, prefer the term "relaxation factor" to the variable name "relaxation_factor".

Reply:
done addressing.